Cargo.toml

```diff
@@ -72,6 +74,7 @@ lazy_static = { version = "1", default-features = false }
 libc = { version = "0.2.105", default-features = false }
 log = { version = "0.4", default-features = false }
 netns-rs = { version = "0.1", default-features = false }
+nix = { version = "0.26.2", default-features = false }
```

Comment:
I for one am very pro `nix` and have 0 objections to using it in `xtask`.
ISTR in any we only used `libc` and I don't think we need to be as strict, but flagging it anyway.

Reply:
Ack.